metadata_registry/settings/secrets_template.py

```diff
@@ -1,13 +1,13 @@
-import ensembl_metadata.settings.env
+import metadata_registry.settings.env
```

Comment:
Not sure about it, but I saw many updates the other way around e.g. from `metadata_registry` to `ensembl_metadata`. IS it done on purpose?

Reply:
Yes, this is right - the project is 'metadata_registry', so the settings live there; 'ensembl_metadata' is the app connected to the `ensembl_metadata` database.

Reply:
ok!